index.html

```diff
@@ -6,7 +6,6 @@
   <title>Document</title>
 </head>
 <body>
-  <h1>Page title</h1>
-  
+  <h1>Hello world!</h1>
 </body>
 </html>
```

Comment:
![image](https://github.com/dsfreedom/project/assets/96409563/fa8fa549-ef2a-4a8b-b7f4-c310a03b8724)
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file